components/FileInput/index.tsx

```diff
@@ -0,0 +1,47 @@
+import React, { ReactElement, useRef } from "react"
+import classNames from "classnames"
+
+export interface FileInputProps
+  extends React.InputHTMLAttributes<HTMLInputElement> {
+  onChangeHandler: (files: File[]) => void
+  inputId: string
+  element: ReactElement
+  isDisabled?: boolean
```

Comment:
`disabled` is already part of the html attributes, would it make sense to just use that?

I think it will be valuable for the API to be as close to a normal input element as possible.

Reply:
It definitely improved the component to use the standard input props 💪 

good input - change applied👍